File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/mediaTrackSelection/IMediaTrackSelection.java
package org.ngbp.jsonrpc4jtestharness.rpc.mediaTrackSelection;

import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcMethod;
import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcType;


@JsonRpcType
public interface IMediaTrackSelection {

    @JsonRpcMethod("org.atsc.track.selection")
    Object mediaTrackSelection();
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/mediaTrackSelection/IMediaTrackSelection.kt
package org.ngbp.jsonrpc4jtestharness.rpc.mediaTrackSelection

import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcMethod
import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcType
import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse

@JsonRpcType
interface IMediaTrackSelection {
    @JsonRpcMethod("org.atsc.track.selection")
    fun mediaTrackSelection(): RpcResponse
}